bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -273,6 +274,7 @@ async def apply_infraction(
                 Member: {messages.format_user(user)}
                 Actor: {ctx.author.mention}{dm_log_text}{expiry_log_text}
                 Reason: {reason}
+                Jump url: {jump_url_text}
```

Comment:
```suggestion
                Jump URL: {jump_url_text}
```

For consistency with the one in `management.py

Reply:
Will do, thanks :+1: .

Reply:
Done (https://github.com/python-discord/bot/pull/2372/commits/ed59a9fd38f8bcfcb81efaf66f86e7ebc6661665).